docs/resources/aws_guardduty_detectors.md

```diff
@@ -0,0 +1,64 @@
+---
+title: About the aws_guardduty_detectors Resource
+platform: aws
+---
+
+# Requirements
+
+Train AWS version 0.1.18 or newer is required for this resource.
+
+# aws\_guardduty\_detectors
+
+Use the `aws_guardduty_detectors` InSpec audit resource to test properties of some or all AWS GuardDuty Detectors.
+
+## Syntax
+
+ Ensure you have exactly 3 Detectors available to you.
+
+    describe aws_guardduty_detectors do
+      its('detector_ids.count') { should cmp 3 }
+    end
+    
+    
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on GuardDuty Detectors](https://docs.aws.amazon.com/guardduty/latest/ug/what-is-guardduty.html).
+
+## Properties
+
+|Property                 | Description|
+| ---                     | --- |
+|detector_ids             | The architecture of the image. |
+
+## Examples
+
+##### Ensure ID of an AMI GuardDuty Detector
+    describe aws_guardduty_detectors do
+      its('detector_ids') { should include ['image-id-43542'] }
```

Comment:
should this actually be checking for `detector-id` rather than `image-id`

Reply:
Updated in latest commit.